lib/version.ts

```diff
@@ -78,6 +78,9 @@ export default async function getDesiredVersion(
   if (versionRange === "latest") {
     console.log("No version specified, using latest");
     return versionInfo.latestVersion;
+  } else if (versionRange.endsWith("-SNAPSHOT")) {
+    console.log("SNAPSHOT version specified");
```

Comment:
I don't think this log is necessary, since later on down the line the downloaded version will be logged with the SNAPSHOT suffix

Reply:
removed logging